<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MostrarLugar</class>
 <widget class="QDialog" name="MostrarLugar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(206, 200, 215);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="title">
    <string>Descripción:</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser_descripcion">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_iniciar_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_atras">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Atrás</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nombre">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jamrul</family>
     <pointsize>36</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 243, 243);
font: 75 36pt &quot;Jamrul&quot;;
color: rgb(73, 60, 107);</string>
   </property>
   <property name="text">
    <string>La Madriguera</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_estrella1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Downloads/iconocalif.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_estrella2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Downloads/iconocalif.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_estrella3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Downloads/iconocalif.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_estrella4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Downloads/iconocalif.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_costo1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Pictures/iconoprecio.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_costo2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Pictures/iconoprecio.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_costo3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Pictures/iconoprecio.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Teléfono:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_telefono">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>33234223423</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>747</x>
     <y>100</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_36">
      <item>
       <widget class="QLabel" name="label_43">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>L</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lunes">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>14:00 - 23:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_37">
      <item>
       <widget class="QLabel" name="label_44">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>M</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_martes">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>14:00 - 23:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_38">
      <item>
       <widget class="QLabel" name="label_45">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>M</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_miercoles">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>14:00 - 23:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_39">
      <item>
       <widget class="QLabel" name="label_46">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>J</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_jueves">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>14:00 - 23:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_40">
      <item>
       <widget class="QLabel" name="label_47">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>V</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_viernes">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>14:00 - 23:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_41">
      <item>
       <widget class="QLabel" name="label_48">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>S</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_sabado">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>Cerrado</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_42">
      <item>
       <widget class="QLabel" name="label_49">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>D</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_domingo">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
        </property>
        <property name="text">
         <string>Cerrado</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_opinion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>590</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(194, 243, 179);</string>
   </property>
   <property name="text">
    <string>Agregar tu valoración</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>641</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Comentarios</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_comentarios">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>615</width>
         <height>173</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_null">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;DejaVu Sans Mono&quot;;
color: rgb(85, 87, 83);</string>
          </property>
          <property name="text">
           <string>No hay comentarios de este negocio</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>70</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
      </property>
      <property name="text">
       <string>De:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
      </property>
      <property name="text">
       <string>A:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_editar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>590</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(218, 221, 151);</string>
   </property>
   <property name="text">
    <string>Editar Informacion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Gargi&quot;;</string>
      </property>
      <property name="text">
       <string>Sucursales:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_sucursales">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>327</width>
         <height>145</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_foto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Pictures/mad.png</pixmap>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>349</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButton_ImagenL">
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ImagenR">
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
  </widget>
  <zorder>splitter_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>pushButton_atras</zorder>
  <zorder>label_nombre</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_opinion</zorder>
  <zorder>pushButton_iniciar_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>layoutWidget_7</zorder>
  <zorder>pushButton_editar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
